src/routes/blog/[[page]]/+page.svelte

```diff
@@ -246,28 +386,28 @@
                             {#if data.currentPage < data.totalPages}
                                 <a
                                     data-sveltekit-noscroll
-                                    class="flex navigation-button"
+                                    class="navigation-button flex"
                                     href="/blog/{data.currentPage + 1}"
                                     class:navigation-button-active={!isLastPage}
                                 >
                                     Next
                                     <span class="web-icon-chevron-right" style="font-size: 20px" />
                                 </a>
                             {:else}
-                                <span class="flex navigation-button">
+                                <span class="navigation-button flex">
                                     Next
                                     <span class="web-icon-chevron-right" style="font-size: 20px" />
                                 </span>
                             {/if}
                         </ul>
                     </div>
-                </div>
+                {/if}
             </div>
-            <div class="pt-[7.5rem]">
-                <div class="container">
-                    <FooterNav />
-                    <MainFooter />
-                </div>
+        </div>
+        <div>
+            <div class="container">
+                <FooterNav />
+                <MainFooter />
             </div>
         </div>
```

Comment:
do we need the `div` around the `div.container`?

Reply:
yes, for the footer's container. else the footer container just squeezes in width.